toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/SceneView.kt

```diff
@@ -156,47 +160,63 @@ public fun SceneView(
-        factory = { sceneView },
-        update = {
-            it.scene = arcGISScene
-            it.interactionOptions = sceneViewInteractionOptions
-            it.labeling = viewLabelProperties
-            it.selectionProperties = selectionProperties
-            it.setTimeExtent(timeExtent)
-            it.cameraController = cameraController
-            it.atmosphereEffect = atmosphereEffect
-            it.spaceEffect = spaceEffect
-            it.sunTime = sunTime
-            it.sunLighting = sunLighting
-            it.ambientLightColor = com.arcgismaps.Color(ambientLightColor.toArgb())
-            it.isAttributionBarVisible = isAttributionBarVisible
-            if (it.graphicsOverlays != graphicsOverlays) {
-                it.graphicsOverlays.apply {
-                    clear()
-                    addAll(graphicsOverlays)
+    Box(modifier = modifier.clipToBounds()) {
+        AndroidView(
+            modifier = Modifier.fillMaxSize().semantics { contentDescription = "SceneView" },
+            factory = { sceneView },
+            update = {
+                it.scene = arcGISScene
+                it.interactionOptions = sceneViewInteractionOptions
+                it.labeling = viewLabelProperties
+                it.selectionProperties = selectionProperties
+                it.setTimeExtent(timeExtent)
+                it.cameraController = cameraController
+                it.atmosphereEffect = atmosphereEffect
+                it.spaceEffect = spaceEffect
+                it.sunTime = sunTime
+                it.sunLighting = sunLighting
+                it.ambientLightColor = com.arcgismaps.Color(ambientLightColor.toArgb())
+                it.isAttributionBarVisible = isAttributionBarVisible
+                if (it.graphicsOverlays != graphicsOverlays) {
+                    it.graphicsOverlays.apply {
+                        clear()
+                        addAll(graphicsOverlays)
+                    }
                 }
-            }
-            if (it.analysisOverlays != analysisOverlays) {
-                it.analysisOverlays.apply {
-                    clear()
-                    addAll(analysisOverlays)
+                if (it.analysisOverlays != analysisOverlays) {
+                    it.analysisOverlays.apply {
+                        clear()
+                        addAll(analysisOverlays)
+                    }
                 }
-            }
-            if (it.imageOverlays != imageOverlays) {
-                it.imageOverlays.apply {
-                    clear()
-                    addAll(imageOverlays)
+                if (it.imageOverlays != imageOverlays) {
+                    it.imageOverlays.apply {
+                        clear()
+                        addAll(imageOverlays)
+                    }
                 }
+            })
+
+        val sceneViewScope = remember(sceneView) { SceneViewScope(sceneView) }
```

Comment:
I believe there is no need to pass a key to remember here:
```suggestion
        val sceneViewScope = remember { SceneViewScope(sceneView) }
```